registry/test.ps1

```diff
@@ -0,0 +1,36 @@
+$ErrorActionPreference = 'Stop'
+
+function addRegistryToDaemonJson() {
+  Write-Host "Adding registry"
+  $daemonJson = "$env:ProgramData\docker\config\daemon.json"
+  $config = @{}
+  if (Test-Path $daemonJson) {
+    $config = (Get-Content $daemonJson) -join "`n" | ConvertFrom-Json
+  }
+  $config = $config | Add-Member(@{ `
+    'insecure-registries' = @("127.0.0.1:5000"); `
+  }) -Force -PassThru
+  $config | ConvertTo-Json | Set-Content $daemonJson -Encoding Ascii
+}
+
+function addPortMapping() {
+  netsh interface portproxy add v4tov4 listenport=5000 listenaddress=127.0.0.1 connectport=5000 connectaddress=$(docker inspect -f '{{ .NetworkSettings.Networks.nat.IPAddress }}' registry)
```

Comment:
This is a workaround for the "Published Ports On Windows Containers Don't Do Loopback" problem. :-)